src/audio/Kconfig

```diff
@@ -338,6 +338,15 @@ config COMP_CODEC_ADAPTER
 	  "src\include\sof\audio\codec_adapter\interfaces.h". It is possible to link several
 	  different codecs and use them in parallel.
 
+config COMP_IGO_NR
+	bool "IGO NR component"
+	default n
+	help
+	  This option enables intelliGo non-speech noise reduction. The propitiatory static library
+	  libigonr.a is needed. The algorithm runs in 16kHz domain with 256 samples in a frame.
+	  GCC is currently not supported because in the library lots of Xtensa intrinsics have been
```

Comment:
typo: proprietary.

Reply:
Does it work on HIFI4? :)

Reply:
thanks. the typo has been fixed.

Currently we only support HiFi3, more specifically `HiFi3 cavs`. The static library is built on that toolchain so SOF image must be built with the same toolchain too.

Reply:
See my comment above on kconfig help text. This needs your company link and contact page so users can aks questions or request new architectures etc.

Reply:
Sure. Thanks for providing a helpful suggestion!